### PR TITLE
feat(endpoints): Create validated movie endpoint and route controller.

### PR DESCRIPTION
**Pull request by virtualdom**
What: Create POST endpoint for movie resource that validates request payload.

Why: Users should be able to add movies to the registry, and the server should be able to expect properly formatted movie inputs.

### CODE REVIEW

File: lib/plugins/features/movies/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      handler: function (request, reply) {
+        reply(Controller.create(request.payload));
+      },
+      validate: {
+        payload: MovieValidator
+      }
+    }
+
+  }]);
+  next();
```
Comment (elnaz) [inline lib/plugins/features/movies/index.js:21]: nit: add an empty line above this

**Comment by elnaz** [comment]
LGTM

File: lib/plugins/features/movies/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    path: '/movies',
+    config: {
+      handler: function (request, reply) {
+        reply(Controller.create(request.payload));
+      },
+      validate: {
+        payload: MovieValidator
+      }
+    }
+
```
Comment (mgartner) [inline lib/plugins/features/movies/index.js]: nit: extra blank line here

File: lib/plugins/features/movies/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }]);
+
+  next();
+
```
Comment (mgartner) [inline lib/plugins/features/movies/index.js:22]: nit: extra blank line here

File: test/plugins/features/movies/controller.test.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie      = require('../../../../lib/models/movie');
+
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
```
Comment (mgartner) [inline test/plugins/features/movies/controller.test.js]: Might be nice in this test to make sure that the controller also saves the release year

**Comment by mgartner** [comment]
LGTM, some minor things

File: lib/plugins/features/movies/index.js (hunk 3 above)
Comment (elnaz) [inline lib/plugins/features/movies/index.js:22]: We're having them [pad the `next` on both sides with a blank line](https://github.com/lob/miyagi/blob/master/06-build-your-own-api/10-endpoint-configuration.md#4-define-the-post-movies-route), at least in miyagi, but I'm not sure if we've really discussed our style preference.

Comment (mgartner) [inline lib/plugins/features/movies/index.js:22]: My bad @virtualdom

Comment (virtualdom) [inline lib/plugins/features/movies/index.js:22]: @mgartner 👌

**Pull request merged**